content/ja/docs/concepts/cluster-administration/addons.md

```diff
@@ -44,6 +45,7 @@ content_type: concept
 ## インフラストラクチャ
 
 * [KubeVirt](https://kubevirt.io/user-guide/#/installation/installation)は仮想マシンをKubernetes上で実行するためのアドオンです。通常、ベアメタルのクラスタで実行します。
+* [node problem detector](https://github.com/kubernetes/node-problem-detector)はLinuxノード上で動作し、システムの問題を[Events](/docs/reference/kubernetes-api/cluster-resources/event-v1/)または[ノードのCondition](/ja/docs/concepts/architecture/nodes/#condition)として報告します。
```

Comment:
```suggestion
* [node problem detector](https://github.com/kubernetes/node-problem-detector)はLinuxノード上で動作し、システムの問題を[Event](/docs/reference/kubernetes-api/cluster-resources/event-v1/)または[ノードのCondition](/ja/docs/concepts/architecture/nodes/#condition)として報告します。
```

I think "Event" would be better as above.

1. The following is the localization rule. (https://kubernetes.io/ja/docs/contribute/localization/)
    ```
    また、日本語では名詞を複数形にする意味はあまりないので、
    英語の名詞を利用する場合は原則として単数形で表現してください。
    ```
3. Linked site's title is "Event".
4. In English's following site, "Event" is used.
    * https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/configuration/manage-resources-containers.md#L200:~:text=be%20found.%20An-,Event%20is,-produced%20each%20time
    * https://github.com/kubernetes/website/blob/main/content/en/docs/reference/labels-annotations-taints/audit-annotations.md#:~:text=objects%20in%20the-,Event%20API,-(API%20group%20events

Reply:
Thanks, fixed.